make-release.sh

```diff
@@ -1,10 +1,22 @@
 #!/bin/bash
 
-# Sets you up to make a release.  Pass in release name, no prefixed `v`.
+# Pass in release name, no prefixed `v`.
+# Make sure that git flow is configured with a version tag prefix of `v`.
 
-if [ -z $1 ]
+export RELEASE_OR_HOTFIX=$1
+VERSION=$2
+
+if [[ $RELEASE_OR_HOTFIX == "release" || ($RELEASE_OR_HOTFIX == "hotfix") ]]
+then
+    echo "You have selected $RELEASE_OR_HOTFIX"
+else
+    echo "usage: You must specify one of release or hotfix. eg $0 release 3.0.0"
+    exit -1
+fi
+
+if [ -z $VERSION ]
 then
-    echo "Pass in release name, no prefixed 'v'."
+    echo "usage: Pass in hotfix name, no prefixed 'v'."
```

Comment:
hmm, mistake originally on my part I guess, but it should say "release/hotfix name"